mappings/src/androidMain/kotlin/com/revenuecat/purchases/kmp/mappings/LogHandler.android.kt

```diff
@@ -0,0 +1,55 @@
+package com.revenuecat.purchases.kmp.mappings
+
+import android.util.Log
+import com.revenuecat.purchases.kmp.LogHandler
+import com.revenuecat.purchases.LogHandler as NativeAndroidLogHandler
+
+public fun NativeAndroidLogHandler.toLogHandler(): LogHandler =
+    (this as LogHandlerWrapper).wrapped
+
+public fun LogHandler.toAndroidLogHandler(): NativeAndroidLogHandler = LogHandlerWrapper(this)
+
+public class DefaultLogHandler: LogHandler {
```

Comment:
Note that I've had to copy the default log handler. This makes sense, since logs are treated differently in Android/Ios and if we want to be able to have logs in the KMP side. Do lmk if you have other thoughts!

Reply:
I don't think this will work, because `DefaultLogHandler` is not a `LogHandlerWrapper`, and so `toLogHandler()` will crash when `Purchases.logHandler` is read. 

I have a proposal here, which avoids the need to copy the `DefaultLogHandler` by wrapping the default from the native Android side if `Purchases.logHandler` is read before it is set: https://github.com/RevenueCat/purchases-kmp/pull/162

Reply:
Resolved with #162.